post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-configuration.adoc

```diff
@@ -8,17 +8,17 @@ toc::[]
 
 An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures.
 
-[NOTE]
-====
-When there are nodes with multiple architectures in your cluster, the architecture of your image must be consistent with the architecture of the node. You need to ensure that the pod is assigned to the node with the appropriate architecture and that it matches the image architecture. For more information on assigning pods to nodes, see link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/[Assigning pods to nodes].
-====
+Configuring multi-architecture compute machines involves some additional considerations:
 
-[IMPORTANT]
-====
-The Cluster Samples Operator is not supported on clusters with multi-architecture compute machines. Your cluster can be created without this capability. For more information, see xref:../../installing/overview/cluster-capabilities.adoc#cluster-capabilities[Cluster capabilities].
-====
+* When there are nodes with multiple architectures in your cluster, the architecture of your image must be consistent with the architecture of the node. You need to ensure that the pod is assigned to the node with the appropriate architecture and that it matches the image architecture. For more information on assigning pods to nodes, see link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/[Assigning pods to nodes].
+
+* If you are planning to manually configure a multi-architecture cluster with a compute node of a second architecture, ensure that your cluster is configured with `platform=none`. You must set this value manually in your `install-config.yaml`, when installing your x86 control plane node.
```

Comment:
This is not true for x86+arm clusters on AWS, Azure, and GCP, we should maintain the platform settings for these deployments.